### PR TITLE
[FIX] 16/17: Proper ~root/.ssh pre-removal

### PR DESCRIPTION
**Pull request by mfloridosygel**
Hello, 

I found that the symlink to /opt/odoo/custom/ssh still being created as sub-directory (/root/.ssh/ssh) so I got an error when I tried to  build my Odoo image.

Docker uses "/bin/sh" as default shell and "[[ ]]" is a specific comparison construct of Bash.

I remove the brackets and now the symlinks to /opt/odoo/custom/ssh is being created succesfully.

Issue origin: https://github.com/Tecnativa/doodba/pull/585

@pedrobaeza

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Great, thanks for the fix.

Good point using a regexp for indicating the versions, but better to be explicit and as brackets are used for separators, it seems weird that way. Please rephrase it to `[FIX] 16/17: Proper ~root/.ssh pre-removal`, and you can put as extra commit message lines the explanation given in this PR.

**Comment by mfloridosygel** [comment]
Done :ok_hand: 

Thank you!

**Comment by pedrobaeza** [comment]
The explanation given in the first comment of the PR (which is pretty good) has not been put in the commit message. It's also better to use a neutral redaction avoiding first person. Something like:

```
[FIX] 16/17: Proper ~root/.ssh pre-removal

The symlink to /opt/odoo/custom/ssh is still being created as sub-directory 
(/root/.ssh/ssh), getting an error when trying to build the Doodba image.

Docker uses "/bin/sh" as default shell and "[[ ]]" is a specific comparison
construct of Bash.

Removing the brackets, now the symlinks to /opt/odoo/custom/ssh are being
successfully created.
```

**Comment by mfloridosygel** [comment]
Hello,

Sorry I did not understand you at the first time.

Now it is done :+1: . 

Thank you!

@pedrobaeza

**Review by ap-wtioit — APPROVED** [review]
Good catch.

**Review by pedrobaeza — APPROVED** [review]
